calc.rb

```diff
@@ -0,0 +1,66 @@
+answer = nil
+puts "Hi there"
+while answer == nil 
+  puts "Please enter one of the following operations, addition, subtraction, \nmultiplication, division, exponent, or modulo:"
+  operation = gets.chomp.downcase
+#gets user input
+  puts "Please enter the first number:"
+  user_num1 = gets.chomp
+  num1 = user_num1.to_f
```

Comment:
Oh I see. I honestly don't remember why I did that, maybe I didn't know about unless instead of using if to test if the answer needed to be a float? I'll change that.